com.itopia.client.json

```diff
@@ -0,0 +1,245 @@
+{
+    "app-id": "com.itopia.client",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "command": "itopia Remote Desktop Client",
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pcsc",
+        /* Needs to talk to the network */
+        "--share=network",
+        /* Play sounds redirected from guests */
+        "--socket=pulseaudio",
+        /* Allow rw access to download folder */
+        "--device=all",
```

Comment:
Is that really required?

Reply:
we need raw usb device access for libusb, is there another way?